lib/jbuilder.rb

```diff
@@ -32,41 +33,12 @@ def self.encode(...)
     new(...).target!
   end
 
-  BLANK = Blank.new
+  BLANK = Blank.new.freeze
+  EMPTY_ARRAY = [].freeze
```

Comment:
There were quite a few spots allocating a new empty array when dealing with an empty collection. Figured it was worthwhile to re-use the same `Array` instance. This does assume that no call sites attempt to mutate the output from `Jbuilder#target!`.